extensions/youtube-downloader/src/utils.ts

```diff
@@ -1,15 +1,15 @@
 import { getPreferenceValues } from "@raycast/api";
 import { formatDuration, intervalToDuration } from "date-fns";
 
-export const preferences = getPreferenceValues<ExtensionPreferences>();
+export const preferences = getPreferenceValues();
 
-export type DownloadOptions = {
+export interface DownloadOptions {
   url: string;
-  format: string;
   copyToClipboard: boolean;
   startTime?: string;
   endTime?: string;
-};
```

Comment:
Can we revert this unrelated change? Changing `type` to `interface` is unnecessary.

I recommend using `type` unless you need declaration merging.